gbi.h

```diff
@@ -1209,17 +1210,17 @@ typedef struct {
 } OcclusionPlane_t;
 
 typedef struct {
-    /* Four vertices of a quad, XYZ components in world space */
     struct {
         short x;
         short y;
         short z;
-    } v[4];
-    float weight; /* Higher if there's a lot of stuff behind it */
+    } v[4]; /** Four vertices of a quad, XYZ components in world space */
```

Comment:
put a doxygen on field without his structure have a doxygen comment will not reference it. But this can be done later